src/controllers/customercontroller.js

```diff
@@ -5,7 +5,6 @@ class CustomerController extends OrderController
         this._loadTemplate(
             "src/views/template/customer.html",
             "#customers",
-            // TODO: there must be something more to display here, perhaps also make a customer class?
```

Comment:
This part causes a merge conflict. Since this part is not actually all that important for your code, I'd advise restoring this to the non-changed situation. (i.e., restore that comment :)).